dask/tests/test_highgraph.py

```diff
@@ -265,3 +265,18 @@ def test_len_does_not_materialize():
     assert len(hg) == len(a) + len(b) == 7
 
     assert not hg.layers["b"].is_materialized()
+
+
+def test_node_tooltips():
+    da = pytest.importorskip("dask.array")
+
+    a = da.ones((1000, 1000), chunks=(100, 100))
+    b = a + a.T
+    c = b.sum(axis=1)
+
+    hg = c.dask
+    g = to_graphviz(hg)
+
+    for layer in g.body:
+        if "label" in layer:
+            assert "tooltip" in layer
```

Comment:
Can you confirm this test fails if you run it on the dask main branch?

And is there a way we can check the length of the information in the tooltip?

Reply:
Yes, it fails on the `main` branch
![image](https://user-images.githubusercontent.com/62539811/128670186-36518758-4800-4b77-9fc5-ef26a34e7580.png)

>  is there a way we can check the length of the information in the tooltip?

I could simply save the length of the tooltip string in the `to_graphviz()` function.
If you want this to be included in the test, I could try some regex.

Reply:
The only reason the tooltip would look super long is the column names in Dataframes and I have taken care of that in the `to_graphviz()` itself.

Reply:
> I could simply save the length of the tooltip string in the `to_graphviz()` function.
> If you want this to be included in the test, I could try some regex.

No, we wouldn't want to save save the length of what the tooltip string should be in the `to_graphviz()` function - the idea of the test is that we check the function did what it was supposed to do by looking at the actual output that gets displayed.

Yes, let's try to use regex to check the length of the node's tooltip string. I'm assuming that part of the string will look something like this: `... tooltip="some tooltip information here" ...`
Then we could use `tooltip="` and the closing quote `"` as the regex start & end markers and check that the number of characters inside the tooltip quotes is not zero.